src/platform/mod.rs

```diff
@@ -68,6 +68,7 @@ macro_rules! impl_platform_host {
         // functions within the callback.
         //
         // TODO: Confirm this and add more specific detail and references.
+        #[must_use = "If the stream is not assigned it will not play."]
```

Comment:
```suggestion
        #[must_use = "If the stream is not stored it will not play."]
```
I think `stored` is a better word. In theory you could also use APIs like `HashMap::insert` or `vec.push` to store the stream, neither I would call assignment.

Reply:
@est31 Changed in [085fa63](https://github.com/RustAudio/cpal/pull/772/commits/085fa6347fc1a8c125a230ffde2cab4f71078e2b) to more closely match https://doc.rust-lang.org/src/std/io/error.rs.html#799

Reply:
I still like stored more. Can you change it to that?

Reply:
@est31 changed in [e158915](https://github.com/RustAudio/cpal/pull/772/commits/e158915e3cb4b656afda928477592e804a51fa43)